src/js/themes/base.js

```diff
@@ -198,7 +198,10 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'focus',
         },
       },
-      font: { ...fontSizing(0) },
+      font: {
+        size: 'inherit',
```

Comment:
Do we need both of these at all in the base theme for this approach? What about omitting them entirely and just not setting them at all?